### ISSUE TITLE
Import domain zone from DNS query

### ISSUE THREAD

**Issue by ghost** [poweradmin/poweradmin#91]
Reported by:    tonix 

It would be extremely useful in migrations to have the possibility of importing a zone (in template or zone) from an existing DNS. So it would be very good to implement a query asking for all items or a multiple single query asking for single elements (www,mx,etc), and then creating a zone from results of queries.

**Comment by dajones70** [comment]
It would be very helpful to be able to take a zone file and do a bulk import of records by pasting them into a large form field.  You don't always have access to setup as a slave, do a zone transfer then change the type to MASTER.
The input data would need to be sanitized then parsed similarly to the zone2sql command (but actually work).

**Comment by artlog** [comment]
Isn't it covered by #986 ? there is a module mentioned in https://github.com/poweradmin/poweradmin/issues/986#issuecomment-3970996221

**Comment by edmondas** [comment]
The import module (#986) only reads a BIND zone file you upload, so you already need the zone exported to a file first.

This request is different: query a live DNS server directly (or AXFR) and build the zone from the response. No file involved. I'd keep it as a separate feature.

**Comment by edmondas** [comment]
I don't think we should build the live-query part ourselves. PowerDNS already handles pulling a zone off a running server: set it up as a slave, let it transfer in, then switch it to MASTER or NATIVE. stasic mentioned this above.

Doing the queries ourselves is worse anyway. Most servers refuse AXFR, and going record by record never gets the full zone. Letting a user type a hostname and having the server query it is also a security risk.

So between the BIND file import in #986 and the slave setup, both cases are already covered.
